src/corehost/cli/runtime_config.cpp

```diff
@@ -71,6 +72,21 @@ bool runtime_config_t::parse_opts(const json_value& opts)
         m_prerelease_roll_fwd = prerelease_roll_fwd->second.as_bool();
     }
 
+    auto roll_fwd_on_no_candidate_fx = opts_obj.find(_X("rollForwardOnNoCandidateFx"));
+    if (roll_fwd_on_no_candidate_fx != opts_obj.end())
+    {
+        m_roll_fwd_on_no_candidate_fx = roll_fwd_on_no_candidate_fx->second.as_bool();
```

Comment:
I think there is a bit of misunderstanding - What I am suggesting is that "m_roll_fwd_on_no_candidate_fx" is maintained as an integer property so that when it is read (vi env var, command line or config), we read the value. Today, we will only support 0 or 1 but in future we could support other >1 values for different policy levels.

Bool property does not help with it. Makes sense?